src/PropertyInfo/TypeInfoParsers/NativeTypeInfoParser.php

```diff
@@ -19,11 +17,8 @@
  *
  * @author Mihai Stancu <stancu.t.mihai@gmail.com>
  */
-abstract class NativeTypeInfoParser implements TypeInfoParserInterface
+trait NativeTypeInfoParser
```

Comment:
Hi @mihai-stancu,

As you've maybe seen, this library will be moved in the Symfony framework. Some majour design issues need to be fixed because the merge, see #8

The work is almost done, the only missing piece is the Hack support. Do you think you can add it in this branch?